### ISSUE TITLE
No screenshot on player

### ISSUE THREAD

**Issue by bottleblue** [YePpHa/YouTubeCenter#1922]
Is it just me or does the screenshot that's supposed to appear on the player when you load/refresh a page not work for HTML5? All I get is a black screen. It works for Flash.

**Comment by ireun** [comment]
I don't get it about what "screenshot" are you talking about..

**Comment by bottleblue** [comment]
Sorry for late reply I didn't get a notification from github. Yes I mean the thumbnail (I thought the thumbnails were only for the tiny pics). I can see it when using Flash player or when using HTML5 on the redesigned player (www.youtube.com/testtube) but not using the default HTML5 player.  
I am using the latest dev script but it's been happening for as long as I can remember. I am using Firefox 39.0. This problem doesn't happen when using HTML5 on other youtube scripts like Youtube+.

**Comment by bottleblue** [comment]
Any suggestions? Is it possibly some setting in YTC that's causing it not to appear?

**Comment by ireun** [comment]
Hmm, maybe some addons are causing this? Try disabling them..

**Comment by bottleblue** [comment]
But I tested it on a new firefox profile with no other addons

**Comment by beefcake3000** [comment]
You have to turn off auto-buffering for the thumbnail in the main player to show up.

**Comment by bottleblue** [comment]
I already had auto-buffering turned off for all the options: Prevent auto-buffering, Prevent playlist auto-buffering, Prevent tab auto-buffer, and Prevent tab playlist auto-buffering. 
I'm currently using the redesigned youtube player (https://www.youtube.com/testtube) which does show the thumbnail but it won't let me switch back to the original player anymore. I guess this is going to be the default player now.

**Comment by beefcake3000** [comment]
I never signed up for the YouTube betas/tests and I have the new player. I think it was widespread update around two weeks ago.

**Comment by bottleblue** [comment]
I guess the problem is solved then since this is the default player. I don't like that it doesn't tell you the length of the video without playing it first.

**Comment by beefcake3000** [comment]
Yes, I agree! That has been bothering me too. Everything else is an improvement so I don't mind it overall.
